app/src/main/java/app/gamenative/service/SteamService.kt

```
@@ -1659,12 +1657,16 @@ class SteamService : Service(), IChallengeUrlChanged {
 
                 try {
                     steamClient!!.servers.tryMark(steamClient!!.currentEndpoint, PROTOCOL_TYPES, ServerQuality.BAD)
+                } catch (e: NullPointerException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)